lib/ql_file.qpp

```diff
@@ -678,12 +679,60 @@ if (mkdir("/tmp/newdir", 0755))
 
     @see umask()
  */
-int mkdir(string path, softint mode = 0777) [dom=FILESYSTEM] {
+int mkdir(string path, softint mode = 0777, bool parents = False) [dom=FILESYSTEM] {
+    const size_t len = strlen(path->getBuffer());
+    char _path[PATH_MAX];
+    char *p;
+
+    errno = 0;
+
 #ifdef _Q_WINDOWS
-   return _mkdir(path->getBuffer());
+    char sep = '\\';
```

Comment:
this is already defined in common.h, please do not redefine - use `QORE_DIR_SEP`